private/aws-util-test/src/requests/test-http-handler.js

```diff
@@ -0,0 +1,117 @@
+export class TestHttpHandler {
```

Comment:
do you need to generate this .js and .d.ts file? can they be removed?

Reply:
Those shouldn't have been committed.